ui/packages/shared/profile/src/ProfileSource.tsx

```diff
@@ -25,8 +25,8 @@ export interface ProfileSelection {
   Type: () => string;
 }
 
-export const timeFormat = 'MMM D, [at] h:mm:s a [(UTC)]';
-export const timeFormatShort = 'MMM D, h:mma';
+export const timeFormat = 'MMM d, [at] h:mm:s a [(UTC)]';
+export const timeFormatShort = 'MMM d, h:mma';
```

Comment:
The time format token is a little different in date-fns https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md